src/providers/fault-store/fault-store.ts

```diff
@@ -66,4 +76,83 @@ export class FaultStoreProvider {
   getTotals() {
     return this.store.getState();
   }
+
+  getTestResult() {
+    let fs = 0;
+    let ds = 0;
+    let ss = 0;
+    const sectionRegEx = new RegExp(/^[a-z]*/);
+    const data = this.getTotals();
+    Object.keys(data.faults).forEach((fault) => {
+      if (fault === 'lastFault') {
```

Comment:
For Alpha, I think this is great but I think the logic for this might be better placed in a reducer, and calculated when a fault change occurs. This way, we would have access to results data and maybe, more importantly, if they have failed during the test, and not just when we call this function. It could also make it easier to reuse this logic if it was placed in its own middleware.